piet-cairo/src/text.rs

```diff
@@ -158,6 +157,10 @@ impl TextLayout for CairoTextLayout {
         self.size
     }
 
+    fn image_bounds(&self) -> Rect {
```

Comment:
Do we want a TODO here that it's approximate? But the entire cairo stack is currently a "toy" so I don't think it matters much.

Reply:
yea, that was my position.